controllers/networking-calico/charts/internal/calico/values.yaml

```diff
@@ -4,11 +4,16 @@ config:
   backend: bird
   ipam:
     type: "host-local"
-#   subnet: "usePodCidr"
+    # subnet: "usePodCidr"
   typha:
     enabled: true
-#ipip: "Always"
-#ipAutodetectionMethod: "first-found"
+  ipv4:
+    pool: ipip # or vxlan
+    mode: "Always"
+    autodetection-method: "first-found"
```

Comment:
Of course we can, but in images we have the hyphens, this is why I thought it would be more uniform not to camel case it. But if you want, I can change this.

Reply:
Changed it to camel-case again, also makes the counter-intuitive `index`ed access in the helm chart unnecessary.